src/app/core/services/connection-status-observer.service.ts

```diff
@@ -25,6 +25,10 @@ export class ConnectionStatusObserver implements OnDestroy {
         })
     }
 
+    public subscribeToClosed(consumer: () => void): EventSubscription {
```

Comment:
This might just be my interpreatation of `consumer`, but I understand "consumer" as the class that wants to consume/use the EventObserver and that the Consumer has an action to be executed. I don't interpret "consumer" as the action/callback to be executed.
What you are parsing here is that "action"/callback, so in my mind `consumer` is not the right word.

You could call the parameter for `callback`, but that is still vauge. Since you method is called `subscribeToClosed` (which postfixing "event" to wouldn't hurt), you could call the parameter for `onClosed`. 
Then your method would look like this:
```
public subscribeToClosedEvent(onClosed: () => void): EventSubScription {
  return this.eventObserver.subscribe(ConnectionEventType.CLOSED, onClosed)
}
```
This makes it quite clear for the "consumer" of `ConnectionStatusObserver` what the consumer needs to parse along to this method.

Reply:
I think that it might be a difference in which paradigm we were "raised". Personally, I see a function as an object for composition and hereby a worthy "consumer" of events. I can get behind the `onClosed` naming scheme.
I have applied it for `RundownEventObserver` and `ConnectionStatusObserver`.